lib/Devel/NYTProf.pm

```diff
@@ -617,11 +617,19 @@ closed.  Calling DB::disable_profile() doesn't do that.  To make a profile file
 usable before the profiled application has completed you can call
 DB::finish_profile(). Alternatively you could call DB::enable_profile($newfile).
 
+You do not need to explicitly C<use Devel::NYTProf> or C<use DB>
+in your perl code in order to call 
+DB::enable_profile(), DB::disable_profile() or DB::finish_profile().
+Those functions are automatically made available to you
+because NYTProf runs as part of the perl debugger framework.
+(Recall that the C<-d> option in C<perl -d:NYTProf some_perl.pl> is 
+the debugger option.)
```

Comment:
The confusion that I was trying to address was that it looked like those functions were being called *without* importing either the DB or Devel::NYTProf packages.  By trial and error I discovered that I was able to call those functions without saying "use Devel::NYTProf" in my code (or "use DB"), which puzzled me.  I then figured out that -- if I'm correct -- it is the "-d" option that is causing DB to be loaded.  So my intent was to alert other readers to the fact that those functions can be called without having any explicit "use" statements to load the DB module.

How about the following?
> You do not need to -- and should not -- put an explicit
> C<use Devel::NYTProf> or C<use DB> in your perl code in
> order to call DB::enable_profile(), DB::disable_profile() or
> DB::finish_profile().  Those functions are automatically loaded
> because NYTProf runs as part of the perl debugger framework.
> (Recall that the C<-d> option in C<perl -d:NYTProf some_perl.pl>
> is the debugger option.)

Feel free to phrase it differently if've still got something wrong or you think a different way of expressing it would be better.   BTW, thanks for a fabulous module!